spec/models/comment_spec.rb

```diff
@@ -31,143 +31,207 @@
       end
     end
 
-    shared_examples "disallows editing the comment if it's changed significantly" do
-      it "prevents editing the comment if it's changed significantly" do
-        subject.edited_at = Time.current
-        subject.comment_content = "Spam content" * 12
-        expect(subject.save).to be_falsey
-        expect(subject.errors[:base]).to include("This comment looks like spam to our system, sorry! Please try again.")
-      end
+    context "when any comment is considered spam" do
```

Comment:
this seems like a misnomer, later down below you have `receive(:spam?).and_return(false)` so not all comments are considered spam. I think it'd be better to have the spam flagging on creation in its own context and the keep `"when any comment is considered spam"` context the same